test/test_torch.py

```diff
@@ -12343,6 +12343,16 @@ def test_memory_format_preserved_after_permute(self, device):
         y = nhwc.permute(0, 1, 3, 2).permute(0, 1, 3, 2)
         self.assertTrue(y.is_contiguous(memory_format=torch.channels_last))
 
+    def test_memory_format_to_operator(self, device):
+        x = torch.randn(4, 3, 8, 8, device=device).contiguous(memory_format=torch.channels_last)
+        z = x.to(device=device)
+        self.assertTrue(z.is_contiguous(memory_format=torch.channels_last))
+        self.assertEqual(x.data_ptr(), z.data_ptr())
```

Comment:
super nitty: it's better to put this line before the one above, because it's clearer what this is meant to test (i.e. when I first read the line above, I thought you were testing that `to` preserves memory_format (which depends on whether we've flipped the switch or not), but this line makes it clear you are testing the case when nothing should change.